server/src/main/java/org/candlepin/policy/js/JsRunner.java

```diff
@@ -116,7 +121,8 @@ Object unwrapReturnValue(Object result) {
             returner = this.invokeMethod(ruleName);
         }
         catch (NoSuchMethodException ex) {
-            log.info("No rule found: " + ruleName + " in namespace: " + namespace);
+            log.info(
+                "No rule found: " + ruleName + " in namespace: " + namespace);
```

Comment:
In the future, when you encounter log output that contains string concatenation in a class you're updating, replace it with substitution: ```log.info("No rule found: {} in namespace: {}", ruleName, namespace);```